packages/terra-horizontal-divider/tests/jest/HorizontalDivider.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import HorizontalDivider from '../../src/HorizontalDivider';
+
+describe('HorizontalDivider', () => {
+  const defaultRender = <HorizontalDivider />;
+
+  // Snapshot Tests
+  it('should render a basic component', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  // Structure Tests
+  it('should have the class divider', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper.prop('className')).toContain('divider');
+  });
+
+  // Custom Prop Test
+  it('it should pass in a custom prop', () => {
+    const wrapper = shallow(<HorizontalDivider id="testDivider" />);
+    expect(wrapper.unrendered.props.id).toEqual('testDivider');
```

Comment:
Why are you checking unrendered props? This could be tested by comparing snapshots.

Reply:
Updated https://github.com/cerner/terra-core/pull/962/commits/9f8a5deba40296bf91341b6ff1d0d56f4bae16ac#diff-46429ad4c25a0b00f25c0054ce0dddf6R21